integration/e2e/config-microservices.tmpl.yaml

```diff
@@ -23,6 +23,7 @@ ingester:
 storage:
   trace:
     blocklist_poll: 2s
+    blocklist_poll_stale_tenant_index: 15s
```

Comment:
What kind of errors does this setting resolve?

Reply:
Since there are no compactors, no component is responsible for writing the tenant index. So the tenant index only gets created once when fetching the tenant index fails (the component falls back to polling and writes the index). If the component making the tenant index does not have a complete view of the blocklist, this tenant index will be wrong and never updated.
Setting the stale duration ensures the query-frontend and querier reject the tenant index after a while.

An alternative solution would be to add a compactor to the system but this might introduce other race conditions as the blocklist length will change due to compactions.

I've added some logs in the description: https://github.com/grafana/tempo/pull/1341/commits/990db134e8275fe2c14da3c20ad15e7905815e01

Reply:
Hmm, this is fine. If there's an option to disable tenant index creation (to just enforce blocklist polling always) then I would prefer that in the e2e tests.

Reply:
Don't think you can disable it, but setting stale at 1s forces components to always fall back to polling.